plugin/storage/grpc/shared/grpc_client.go

```diff
@@ -61,7 +61,7 @@ func (c *grpcClient) GetTrace(ctx context.Context, traceID model.TraceID) (*mode
 	trace := model.Trace{}
 	for received, err := stream.Recv(); err != io.EOF; received, err = stream.Recv() {
 		if err != nil {
-			return nil, errors.Wrap(err, "grpc stream error")
+			return nil, spanstore.ErrTraceNotFound
```

Comment:
This doesn't look right, why are you assuming that EVERY error from gRPC is "trace not found"?

Reply:
fixed.